tools/client/README.md

```diff
@@ -19,3 +17,11 @@ If you want to use Sample TEE attester in CC-KBC, set the following environment
 ```
 export AA_SAMPLE_ATTESTER_TEST=yes
 ```
+
+## Client SDK
+
+Add the following dependencies to the `Cargo.toml` of the Rust project to import KBS Client SDK:
+
+```
+kbs_client = { git = "https://github.com/confidential-containers/kbs" }
+```
```

Comment:
I'd prefer to have that one as a separate PR.

Reply:
Okay, I'll separate it.